### PR TITLE
Add perf xml template

### PR DESCRIPTION
**Pull request by ruomengh**
1. Add VM type VM_TYPE_TD_PERF, VM_TYPE_EFI_PERF, VM_TYPE_LEGACY_PERF, referring to perf xml templates.
2. Change "io" and "cache" fields to placeholder in xml template. It can be customized when creating VMsfor FIO test. The default value for "io" and "cache" are "native" and "none".

Signed-off-by: Hao,Ruomeng <ruomeng.hao@intel.com>

### CODE REVIEW

File: utils/pycloudstack/pycloudstack/vmm.py [hunk 1]
```diff
@@ -131,7 +131,10 @@ class VMMLibvirt(VMMBase):
         VM_TYPE_LEGACY: "legacy-base",
         VM_TYPE_EFI: "ovmf-base",
         VM_TYPE_TD: "tdx-base",
-        VM_TYPE_SGX: "sgx-base"
+        VM_TYPE_SGX: "sgx-base",
+        VM_TYPE_TD_PERF: "tdx-base-perf",
+        VM_TYPE_EFI_PERF: "ovmf-base-perf",
+        VM_TYPE_LEGACY_PERF: "legacy-base-perf"
```
Comment (LeiZhou-97) [inline utils/pycloudstack/pycloudstack/vmm.py:137]: Do we need legacy-base-perf? So far, it seems that we only use TD and OVMF in performance test.

**Comment by LeiZhou-97** [comment]
Does second disk support will be added in next PR? I do not see the second disk on current xml template.

**Comment by ruomengh** [comment]
> Does second disk support will be added in next PR? I do not see the second disk on current xml template.

Yes. It's decided to make second disk "FIO specific" but not for all perf tests. The changes will be in the next PR.

**Comment by LeiZhou-97** [comment]
LGTM

**Comment by jialeif** [comment]
Please refer to https://github.com/intel/tdx-tools/commit/87830753ee0b5a0377576def57d6950ff578f818 @LeiZhou-97  defined bind_cpuids

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**